lib/resty/worker/events.lua

```diff
@@ -257,7 +257,7 @@ end
 -- @param data the data for the event, anything as long as it can be used with cjson
 -- @param unique a unique identifier for this event, providing it will make only 1
 -- worker execute the event
--- @return results from the call to `poll`, or nil+error
+-- @return results from the call to `post_event`, or nil+error
```

Comment:
`post_event` is a local undocumented function, so should not be named here. It returns nil+error, or the event_id. So imo it should return `true`